dev-libs/libdivecomputer/libdivecomputer-0.6.0-r1.ebuild

```diff
@@ -29,5 +29,5 @@ src_prepare() {
 }
 
 src_configure() {
-	econf $(use_with bluetooth bluez)
+	econf $(use_with bluetooth bluez) $(use_enable static-libs static)
```

Comment:
Just `--disable-static` rather than add USE static-libs.

Reply:
Anything else won't be accepted